packages/visual-editor/src/components/Dropzone/useComponent.ts

```diff
@@ -63,11 +69,12 @@ export const useComponent = ({ node: rawNode, resolveDesignValue }: ComponentPar
     node,
     areEntitiesFetched,
     resolveDesignValue,
+    renderDropzone,
     definition: componentRegistration.definition,
   });
 
   // Only pass editor props to built-in components
-  const { editorMode, renderDropzone, ...componentProps } = props;
+  const { editorMode, renderDropzone: _renderDropzone, ...componentProps } = props;
```

Comment:
I think `renderDropzone` is destructured from props here so all the remaining props that we want to keep are spread into the `componentProps` variable (which does get used).  This change confused me at first because I wasn't seeing anywhere that `_renderDropzone` gets used.  Do you think this change is necessary?

Reply:
Yes, it's the same for `editorMode`. We don't want to pass them to all kind of components as this creates warnings in the console IIRC.

Reply:
Problem being here that we define `renderDropzone` already before, so I had to rename it :/

Reply:
Got it, that makes sense now